force_bdss/io/workflow_writer.py

```diff
@@ -93,9 +93,6 @@ def traits_to_dict(traits_obj):
     traits version."""
```

Comment:
I suppose we could lose this function now things are done in the `__getstate__` method itself

Reply:
Done